lib/new-settings-loader.js

```diff
@@ -1,71 +1,115 @@
 const _ = require('lodash/fp')
-const low = require('lowdb')
-const FileAsync = require('lowdb/adapters/FileAsync')
+const db = require('./db')
 
-const adapter = new FileAsync('db.json')
-let db = null
-
-// TODO new-admin save to actual db, like the old settings
-low(adapter).then(it => {
-  db = it
-})
+const NEW_SETTINGS_LOADER_SCHEMA_VERSION = 2
 
+// TODO: validate
```

Comment:
We don't want to leave TODOs on our code base, is there a reason for these here? What needs to be validated?

If this is referring to the "valid" row we won't need it anymore, new admin should work in a way that the user never inputs a invalid config.

Reply:
Yeah, it refers to a valid row. I'll remove that, shouldn't be commited.